src/app/Card.tsx

```diff
@@ -178,28 +211,53 @@ const SaveFileButton = ({ cardID }: { cardID: string }) => {
         hover ? 'visible' : 'invisible'
       }`}
       >
-        <div onClick={() => handleItemClick('CSV', 'csv')} className={itemClasses}>
+        <div
```

Comment:
Sorry for all this noise... it's an auto-formatting thing, I tried removing it but couldn't.